gtsfm/densify/mvs_patchmatchnet.py

```diff
@@ -18,9 +18,12 @@
 from gtsfm.common.image import Image
 from gtsfm.densify.mvs_base import MVSBase
 from gtsfm.densify.patchmatchnet_data import PatchmatchNetData
+from gtsfm.evaluation.metrics import GtsfmMetric, GtsfmMetricsGroup
 from gtsfm.utils import logger as logger_utils
 from thirdparty.patchmatchnet.models.net import PatchmatchNet
 
+METRICS_GROUP = "densify_patchmatchnet"
```

Comment:
let's just call the metrics group "multi_view_stereo"? I see PatchmatchNet as an implementation detail / instance of the method. If we use another net instead, I think we would want to keep the name oriented around MVS

Reply:
resolved.